docs/faq.md

```diff
@@ -20,7 +20,18 @@ Linux wheels are built in [`manylinux`/`musllinux` containers](https://github.co
 
 ### Building macOS wheels for Apple Silicon {: #apple-silicon}
 
-`cibuildwheel` supports cross-compiling `universal2` and `arm64` wheels on `x86_64` runners. With the introduction of Apple Silicon, you now have several choices for wheels for Python 3.8+:
+`cibuildwheel` supports cross-compiling `universal2` and `arm64` wheels on `x86_64` runners.
+
+!!! note
+    Currently it is not fully supported to cross-compile project that uses Poetry as a build backend.
+    Reladed  issue [here](https://github.com/python-poetry/poetry/issues/7107).
+    Some packages can be built with arm64 wheels, but their file names will be incorrect,
+    with the platform tag showing `x86_64` instead of `arm64`.
+    As a workaround, the file can be renamed before running delocate to repair the wheel.
```

Comment:
The internal tags will still be broken? You'd need https://github.com/pypa/wheel/pull/422 I believe.

Reply:
I'd remove this renaming suggestion and just say the tags have to be corrected. An example is in https://github.com/scikit-build/cmake-python-distributions/blob/master/scripts/repair_wheel.py.

Reply:
Now better?